index.html

```diff
@@ -57,6 +57,329 @@ <h2>I'm a Software Developer</h2>
+              <h4>Frameworks</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">React.js</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSpec</a></li>
+                <li><a href="#">Capybara</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="lang-container">
+            <div class="lang-img">
+              <img src="./pics/rec3.svg" alt="rec" />
+              <h4>skills</h4>
+            </div>
+            <div>
+              <ul class="lang-list">
+                <li><a href="#">Database Management</a></li>
+                <li><a href="#">Version Control</a></li>
+                <li><a href="#">CLI</a></li>
+                <li><a href="#">API Design</a></li>
+                <li><a href="#">Web Design</a></li>
+              </ul>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="interest">
+        <div class="intrest-description">
+          <h5>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h5>
+        </div>
+        <form
+          class="contact"
+          action="https://formspree.io/f/xzbqggzq"
+          method="POST"
+        >
+          <div class="contact-input">
+            <input
+              type="text"
+              placeholder="Full Name"
+              name="name"
+              maxlength="30"
+              required
+            />
+          </div>
+          <div class="contact-input">
+            <input
+              type="email"
+              placeholder="Email Address"
+              name="email"
+              required
+            />
+          </div>
+          <div class="info">
+            <textarea
+              placeholder="Write something ..."
+              rows="6"
+              name="message"
+              required
+              maxlength="500"
+            ></textarea>
+          </div>
+        </form>
+        <div class="contact-btn">
+          <button class="story-contact-btn" type="submit">
+            <a href="#">Get in touch</a>
+          </button>
+        </div>
+      </div>
     </section>
   </body>
+  <footer>
+    <div class="footer-content">
+      <ul class="social-media footer-social">
+        <li>
+          <a href="#"><img src="./pics/github.svg" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/link.svg" alt="linkedin" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/yah.svg" alt="yep" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/twitter.svg" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./pics/medium.svg" alt="medium" /></a>
+        </li>
+      </ul>
+      <hr />
+    </div>
+  </footer>
```

Comment:
As requested by the previous CR please change the background of the footer section to white and remember only the background color is applied to the `textarea element` with the value of the following `background: #FBF8F7;`
`